### ISSUE TITLE
Issue with Webpack Build

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/dustyjs#1]
Seems like when you import the module on an app thats being built out by webpack the module returns an undefined. 

So

```js
import dusty from 'dusty-fns';

console.log(dusty); // => is undefined
```

However doing this

```js
import isObject from 'dusty-fns/isObject';

console.log(isObject); // => returns the function
```

Will be digging into this for a `v0.2.2` patch.

**Comment by dhershman1** [comment]
Fixed with release [v0.2.2](https://github.com/dhershman1/dustyjs/releases/tag/v0.2.2)

**Comment by dhershman1** [comment]
Doesn't look like the patch worked as intended because I am a dumb.
